src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/InviteUserSpec.scala

```diff
@@ -7,8 +7,6 @@ import org.broadinstitute.dsde.workbench.sam.service.GenEmail.genBadChar
 import org.broadinstitute.dsde.workbench.sam.service.{CloudExtensions, TestUserServiceBuilder}
 import org.broadinstitute.dsde.workbench.sam.util.SamRequestContext
 import org.mockito.ArgumentCaptor
-import org.mockito.ArgumentMatchers.any
-import org.mockito.Mockito.verify
```

Comment:
These failed in my compiles because they are unused. I don't know how they slipped into develop.